webservices/flow.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+import networkx as nx
+
+here, _ = os.path.split(__file__)
+home = os.path.join(here, os.pardir)
+script_path = os.path.join(home, 'data', 'sql_updates')
+
+def get_graph():
+    """Build a `DiGraph` that captures dependencies between database migration
+    tasks. Each node represents a migration script, and each edge represents
+    a dependency between tasks. Tasks can be ordered using topological sort.
+    """
```

Comment:
@jmcarp: Just a quick point of order, if we add or modify the SQL scripts that need to be managed by this, we'll have to update the graph here as well, correct?  Just considering where we might want to make a note of that for other developers who come into the project in the future.